lib/ansible/modules/database/oracle/oracle_tablespace.py

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/python
+#  -*- coding: utf-8 -*-
+
+# Copyright (c) 2017 Thomas Krahn (@Nosmoht)
+# All rights reserved.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = '''
+module: oracle_tablespace
+short_description: Manage Oracle tablespaces and datafiles
+description:
+- Create, alter or delete Oracle tablespaces.
+- Add and modify datafiles.
+- All datafiles of a tablespace will be handled equal!
+- For example, if I(init_size) is set the module will ensure the same value for all data files of a tablespace.
+options:
+  name:
+    description:
+    - Tablespace name
+    required: true
+  num_datafiles:
+    description:
+    - Amount of datafiles to ensure within the tablespace.
+    - If current amount is lower than specified new datafiles will be added until I(num_datafiles) are reached.
+    - Datafiles will not be removed if more do exist.
+    required: false
+    default: 1
+  autoextend:
+    description:
+    - Boolean to define if autoextend of datafiles is enabled or disabled.
+    required: false
+    default: false
```

Comment:
Please add `type: bool` (this is the only `type:` that's supported in DOCUMENTATION.

http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#documentation-block